### PR TITLE
Merge 4.7.0 into master

### PR DESCRIPTION
**Pull request by yenienserrano**
Merge 4.7.0 into master

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +50,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Migrate the timeFilter, metaFields, maxBuckets health checks inside the pattern check. [#5384](https://github.com/wazuh/wazuh-kibana-app/pull/5384)
 - Changed the query to search for an agent in `management/configuration`. [#5485](https://github.com/wazuh/wazuh-kibana-app/pull/5485)
 - Changed the search bar in management/log to the one used in the rest of the app. [#5476](https://github.com/wazuh/wazuh-kibana-app/pull/5476)
+  <<<<<<< HEAD
+  =======
```
Comment (Desvelao) [inline CHANGELOG.md:54]: **nitpick**: I think the conflict was not correctly solved.

File: CHANGELOG.md [hunk 2]
```diff
@@ -79,6 +98,12 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Fixed the rendering of tables that contains IPs and agent overview [#5471](https://github.com/wazuh/wazuh-kibana-app/pull/5471)
 - Fixed the agents active coverage stat as NaN in Details panel of Agents section [#5490](https://github.com/wazuh/wazuh-kibana-app/pull/5490)
+  <<<<<<< HEAD
+  =======
```
Comment (Desvelao) [inline CHANGELOG.md:102]: **nitpick**: I think the conflict was not correctly solved.

File: CHANGELOG.md [hunk 3]
```diff
@@ -88,6 +113,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Changed method to perform redirection on agent table buttons [#5539](https://github.com/wazuh/wazuh-kibana-app/pull/5539)
 - Changed windows agent service name in the deploy agent wizard [#5538](https://github.com/wazuh/wazuh-kibana-app/pull/5538)
+  <<<<<<< HEAD
+  =======
```
Comment (Desvelao) [inline CHANGELOG.md:117]: **nitpick**: I think the conflict was not correctly solved.

File: CHANGELOG.md (hunk 2 above)
Comment (yenienserrano) [reply CHANGELOG.md:99]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (yenienserrano) [reply CHANGELOG.md:114]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (yenienserrano) [reply CHANGELOG.md]: Done

**Pull request merged**